IrohaCrypto/Classes/IRMnemonicCreator.m

```diff
@@ -0,0 +1,243 @@
+//
+//  IRMnemonic.m
+//  IrohaCrypto
+//
+//  Created by Ruslan Rezin on 15/11/2018.
+//
+
+#import "IRMnemonicCreator.h"
+#import <CommonCrypto/CommonCrypto.h>
+#import "IRWordList.h"
+
+static const unsigned char BITS_PER_BYTE = 8;
+static const unsigned char STRENGTH_STEP = 32;
+static const unsigned char BITS_PER_WORD = 11;
```

Comment:
why 11?

Reply:
This mean that we use 11 consecutive bits from the entropy per word index for vocabulary with 2^11 size.

Reply:
just from the first appearance it is not clear what it does. you probably need to write short doc